rocAL_pybind/examples/rocAL_api_numpy_reader.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
Looks like there's no division operation being used in this file. Please remove it if it is not used.